### ISSUE TITLE
Help for `round` contains formatting codes?

### ISSUE THREAD

**Issue by cormullion** [JuliaLang/julia#16208]
The help text for `round()` looks odd in the REPL, with formatting codes and numerous backslashes. Is this normal — it doesn't appear for many other help requests...?

```
help?> round
search: round RoundUp RoundDown RoundToZero RoundingMode RoundNearest RoundFromZero RoundNearestTiesUp RoundNearestTiesAway

  .. round(z, RoundingModeReal, RoundingModeImaginary)

  Returns the nearest integral value of the same type as the complex-valued
  ``z`` to ``z``, breaking ties using the specified :obj:`RoundingMode`\ s.
  The first :obj:`RoundingMode` is used for rounding the real components while
  the second is used for rounding the imaginary components.

  ..  round([T,] x, [digits, [base]], [r::RoundingMode])

  ``round(x)`` rounds ``x`` to an integer value according to the default
  rounding mode (see :func:`get_rounding`), returning a value of the same type as
  ``x``. By default (:obj:`RoundNearest`), this will round to the nearest
  integer, with ties (fractional values of 0.5) being rounded to the even
  integer.
```

Oops, forgot `versioninfo()`:

```
julia> versioninfo()
Julia Version 0.4.5
Commit 2ac304d (2016-03-18 00:58 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) i5-2500S CPU @ 2.70GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

**Comment by tkelman** [comment]
`versioninfo()` ?

I suspect this is fixed on master, one of the rst docstrings that wasn't easy to backport the migration for. I'm not sure whether the markdown cross-reference syntax that we use now is supported by 0.4's doc system.

**Comment by stevengj** [comment]
Seems to be fixed in master, where I get:
![image](https://cloud.githubusercontent.com/assets/2913679/15045618/868ef466-12a9-11e6-85a1-384cf74ae89f.png)

**Comment by cormullion** [comment]
Fixed on 0.5, although not on 0.4.7. So I think I'll close it.
